### PR TITLE
[README] Provide Arch Linux installation instructions

### PR DESCRIPTION
**Pull request by frol**
P.S. This project is a great initiative! Good luck on your way!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,17 @@ OWL is not affiliated with or endorsed by Apple Inc. Use this code at your own r
...
 
 
-## Build
+## Installation
+
+If you're an Arch Linux user, then you can install `owl` from the
+[`owl-git` AUR package](https://aur.archlinux.org/packages/owl-git/), e.g.,
+
+```sh
+$ yay -S owl-git
```
Comment (schmittner) [inline README.md]: Remove `$` to enable copy&paste

Comment (frol) [reply README.md]: @schmittner Done

**Comment by Morganamilo** [comment]
These are not really "Arch Linux install instructions" but rather specific to yay. Arch only officially supports makepkg. So I would suggest you base the instructions around makepkg instead.

**Comment by frol** [comment]
@Morganamilo It is there just to, basically, inform people that the package is available in AUR. Feel free to submit a PR changing the text to your liking. I have seen a number of github projects referring to `yay`.

**Pull request merged**